stores/stores.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // SopsFile is a struct used by the stores as a helper to unmarshal the SOPS metadata
 type SopsFile struct {
-	Metadata Metadata `yaml:"sops" json:"sops"`
+	Metadata *Metadata `yaml:"sops" json:"sops"`
```

Comment:
uh?

Reply:
So the JSON and YAML parsers will default the field to the zero value if it's not in the YAML/JSON document. Using a pointer is cleaner imo because it will leave it nil, whereas if we don't use a pointer it'll leave it to the zero value of `Metadata`.

But we could also leave it as-is and then do `if file.Metadata == Metadata{} { ... }`.

Reply:
I'm fine with the change, but it raised an eyebrow so I think you should put that explanation in a comment above, so we don't wonder why it's done this way next time around.